src/main/webapp/styles/main.css

```diff
@@ -139,11 +172,188 @@ img {
 
 #header-logo {
     float: left;
-    width:150px;
+    width: 150px;
     display: inline;
     margin-top: 25px;
 }
 
 .highlight-span {
     color: var(--primaryDark);
+}
+
+
+#appointments {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+#appointments td, #appointments th {
+    border: 1px solid #ddd;
+    padding: 8px;
+    text-align: center;
+}
+
+#appointments th {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    text-align: left;
+    background-color: #e6f3ff;
+    color: black;
+    text-align: center;
+    position: sticky;
+    top: 0;
+    z-index: 2
+}
+#slots {
+    text-align: center;
+    width: 50%;
+    padding: 10px;
+    font-size: 20px;
+    border-radius: 2px;
+}
+
+#options {
+    border-radius: 45%;
+    width: 110px;
+    height: 100px;
+    outline: 0;
+    font-size: 15px;
+    background-color: #e6f3ff;
+}
+
+.blueAlternateButton {
+    color: var(--secondaryLight);
+    background-color: var(--primaryDark);
+    font-weight: bold;
+    border-radius: 12px;
+    border: none;
+    width: 196px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    margin-top: 25px;
+}
+
+.pinkAlternateButton {
+    color: var(--secondaryLight);
+    background-color: var(--alternativeLight);
+    font-weight: bold;
+    border-radius: 12px;
+    border: none;
+    width: 196px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    margin-top: 25px;
+}
+
+#options:hover {
+    background-color: #e31566;
```

Comment:
If we reference colours we should use the variables at the top of the file. Like:

`color: var(--secondaryDark);`

There's a few that need to be updated in this file.